rust/operator-binary/src/controller.rs

```diff
@@ -352,6 +362,7 @@ pub fn build_metastore_role_service(hive: &HiveCluster) -> Result<Service> {
 /// The rolegroup [`ConfigMap`] configures the rolegroup based on the configuration given by the administrator
 fn build_metastore_rolegroup_config_map(
     hive: &HiveCluster,
+    resolved_product_image: &ResolvedProductImage,
```

Comment:
Nit: I just noticed that I put `resolved_product_image` as the last argument in the list. Not sure what is better. Just wanted to mention it :)
Applies to other fn as well

Reply:
Yeah i like it second after the cluster tbh. This will be the same in every operator and every method. The other parameters change... that were my thoughts. What do you think?

Reply:
Works for me. Feel free to change it in the already touched operators ;)